test/autotest.py

```diff
@@ -556,6 +560,7 @@ def testRestart():
       printFixed("ckpt:")
       # NOTE:  If this faile, it will throw an exception to CheckFailed
       #  of this function:  testRestart
+      sleep(S*SLOW)
       testCheckpoint()
```

Comment:
Let's put the following two statements right after this call to `testCheckpoint`:
```
sleep(S*SLOW)
CHECK(doesStatusSatisfy(getStatus(), status))
```